rules-xml/api/src/main/java/org/jboss/windup/rules/apps/xml/condition/XmlFile.java

```diff
@@ -574,7 +574,7 @@ public Object evaluate(@SuppressWarnings("rawtypes") List args) throws XPathFunc
                     }
                 }
 
-                for (Map.Entry<String, String> entry : paramMatchCache.getVariables(frameIdx).entrySet())
+                for (Map.Entry<String, String> entry : paramMatchCache.getVariables().entrySet())
```

Comment:
In looking at the code, I'm not sure either. I think removing it is probably the right thing to do.